package.json

```diff
@@ -19,14 +19,16 @@
     "prebuild:clean": "npx expo prebuild --clean --no-install",
     "prebuild": "npx expo prebuild --no-install",
     "update": "node scripts/push-update.js",
-    "select-images": "node scripts/select-images.js"
+    "select-images": "node scripts/select-images.js",
+    "verify-assets:android": "npx expo export --dump-assetmap -p android; npx expo-updates assets:verify --build-manifest-path android/app/build/intermediates/assets/release/app.manifest --exported-manifest-path dist/metadata.json --asset-map-path dist/assetmap.json --platform android",
```

Comment:
we may also want to prepend this with a `gradlew assembleRelease` or something so that the build intermediates are guaranteed to be present and current

Reply:
If you rebuild, and then do EAS update with no code changes, you are guaranteed to have all assets and the `assets:verify` command will always return 0 . So we just remind the user that a build must have already been created (see the changes in `ASSETSELECTION.md`)